fe/fe-core/src/main/java/com/starrocks/catalog/TableFunctionTable.java

```diff
@@ -316,7 +316,7 @@ private void parseFiles() throws DdlException {
         }
 
         if (fileStatuses.isEmpty()) {
-            throw new DdlException("no file found with given path pattern: " + path);
+            throw new DdlException(String.format("No source file in the path: [%s]", path));
```

Comment:
Put the error message into ErrorCode.java.

Reply:
ok